test/support/models.rb

```diff
@@ -215,7 +215,7 @@ class ValidatingUser < User
     only_integer: true
   validates_length_of :name, maximum: 25
   validates_length_of :description, maximum: 50
-  validates_length_of :action, maximum: 10, tokenizer: lambda { |str| str.scan(/\w+/) }
+  validates_length_of :words_in_action, maximum: 10
```

Comment:
I mean, this test now doesn't make any sense:

https://github.com/plataformatec/simple_form/blob/81b9d7d3439225c7613e570afb44596acfa97a76/test/inputs/string_input_test.rb#L35-L38

We should remove the test when running with Rails 5 but keep for the others versions. The same with this code, we should only define this validation when it is not Rails 5. We should also conditionally define https://github.com/plataformatec/simple_form/blob/81b9d7d3439225c7613e570afb44596acfa97a76/lib/simple_form/components/maxlength.rb#L29-L31.